### ISSUE TITLE
Method with implementation

### ISSUE THREAD

**Issue by ignatandrei** [Frederik91/MakeInterface#32]
I have put your project on my list of Rolsyn Code Generators 

https://ignatandrei.github.io/RSCG_Examples/v2/docs/MakeInterface.Generator

However , I have discovered a bug

If in the class I put
```csharp
public string FullName() =>  $"{FirstName} {LastName}";
```

the interface is generating the same - with implementation!

( In my example , that works, I have put
```csharp
public string FullName()
    {
        return $"{FirstName} {LastName}";
    }
```

and it works!)

**Comment by Frederik91** [comment]
Hi, thanks for adding it to the list! 

I have created a new version with a fix for this (1.4.1), thanks for letting me know :)

I also have another source generator for Dependency Injection that you might want to check out.
[DependencyInjection.SourceGenerator](https://github.com/Frederik91/DependencyInjection.SourceGenerator)

**Comment by ignatandrei** [comment]
Please let me know when it is implemented with IIncrementalGenerator , instead of ISourceGenerator ...
